science/orthanc/Makefile

```diff
@@ -0,0 +1,57 @@
+# Created by: maintainer.freebsd@xpoundit.com
+
+PORTNAME=	orthanc
+DISTVERSION=	1.9.3
+CATEGORIES=	science
+MASTER_SITES=	https://www.orthanc-server.com/downloads/get.php?path=/orthanc/
+DISTNAME=	Orthanc-${PORTVERSION}
+
+MAINTAINER=	maintainer.freebsd@xpoundit.com
+COMMENT=	Orthanc is a DICOM server for healthcare and medical research
```

Comment:
I think we can drop "Orthanc is a" part.

Reply:
The old COMMENT was `RESTful DICOM server for healthcare and medical research`
https://svnweb.freebsd.org/ports/head/science/orthanc/Makefile?revision=471770&view=markup&pathrev=471893